chromium_src/content/child/runtime_features.cc

```diff
@@ -7,8 +7,6 @@ namespace content {
 void SetRuntimeFeaturesDefaultsAndUpdateFromArgs(
     const base::CommandLine& command_line) {
   SetRuntimeFeaturesDefaultsAndUpdateFromArgs_ChromiumImpl(command_line);
-
-  WebRuntimeFeatures::EnableAutoplayMutedVideos(false);
```

Comment:
cc @darkdh

Reply:
addressed here https://github.com/brave/brave-core/pull/358/files#diff-f45e2eb337d4abc3431ff908952a36c6R10
for this upstream commit
https://chromium.googlesource.com/chromium/src/+/49bb4c724406e9095603ad0b121c467ecd746474